docs/admin/pool_operations.md

```diff
@@ -191,174 +191,382 @@ said to be operating in "phase-2" mode.
 --------------------------------------------------
-  UUID             : bdbef091-f0f8-411d-8995-f91c4efc690f
-  Service Leader   : 1
-  Service Ranks    : [0-1]
-  Storage Ranks    : [0-1]
-  Total Size       : 935 GB
-  Metadata Storage : 135 GB (68 GB / rank)
-  Data Storage     : 800 GB (400 GB / rank)
-  Memory File Size : 68 GB (34 GB / rank)
-
-$ dmg pool query bob
-
-Pool 47060d94-c689-4981-8c89-011beb063f8f, ntarget=32, disabled=0, leader=0, version=1, state=Ready
-Pool health info:
-- Rebuild idle, 0 objs, 0 recs
-Pool space info:
-- Target count:32
-- Total memory-file size: 70 GB
-- Metadata storage:
-  Total size: 140 GB
-  Free: 131 GB, min:4.1 GB, max:4.1 GB, mean:4.1 GB
-- Data storage:
-  Total size: 800 GB
-  Free: 799 GB, min:25 GB, max:25 GB, mean:25 GB
-
-$ dmg pool query bob2
-
-Pool bdbef091-f0f8-411d-8995-f91c4efc690f, ntarget=32, disabled=0, leader=1, version=1, state=Ready
-Pool health info:
-- Rebuild idle, 0 objs, 0 recs
-Pool space info:
-- Target count:32
-- Total memory-file size: 68 GB
-- Metadata storage:
-  Total size: 135 GB
-  Free: 127 GB, min:4.0 GB, max:4.0 GB, mean:4.0 GB
-- Data storage:
-  Total size: 800 GB
-  Free: 799 GB, min:25 GB, max:25 GB, mean:25 GB
+  UUID             : b24df7a5-17d5-4e87-9986-2dff18078b6e
+  Service Leader   : 0
+  Service Ranks    : 0
+  Storage Ranks    : 0
+  Total Size       : 1.5 TB
+  Metadata Storage : 151 GB (151 GB / rank)
+  Data Storage     : 1.3 TB (1.3 TB / rank)
+  Memory File Size : 151 GB (151 GB / rank)
 ```
 
-The following examples are with a single host with dual engines where bdev
-roles WAL, META and DATA are shared.
+Looking at this output and comparing with example no. 1 we observe that
+because both SSDs are sharing META and DATA roles, more capacity is available
+for DATA.
+
 
-Single pool with VOS index file size equal to the meta-blob size (`--mem-ratio
-100%`).
+5. If the `--mem-ratio` is then reduced to 50% in the above example, we end up
+with double the Metadata-Storage size which detracts from the DATA capacity.
 
 ```bash
-$ dmg pool create bob --size 100% --mem-ratio 100%
+$ dmg -i pool create bob -z 100% --mem-ratio 50%
+
+Creating DAOS pool with 100% of all storage
+Pool created with 20.32%,79.68% storage tier ratio
+--------------------------------------------------
+  UUID             : 2b4147eb-ade3-4d76-82c4-b9c2c377f8d1
+  Service Leader   : 0
+  Service Ranks    : 0
+  Storage Ranks    : 0
+  Total Size       : 1.5 TB
+  Metadata Storage : 303 GB (303 GB / rank)
+  Data Storage     : 1.2 TB (1.2 TB / rank)
+  Memory File Size : 151 GB (151 GB / rank)
+```
 
-Pool created with 5.93%,94.07% storage tier ratio
+META has been doubled at the cost of DATA capacity.
+
+
+6. Adding another engine/rank on the same host results in more than double DATA
+capacity because RAM-disk capacity is halved across two engines/ranks on the same
+host and this results in a reduction of META and increase in DATA per-rank sizes.
```

Comment:
NIT: an explanation on why total Memory File Size (i.e. sum of the two ranks) is lower than the one engine configuration, could be helpful.

Reply:
fixed